docs/project/refpack-assemblies.md

```diff
@@ -0,0 +1,71 @@
+# List of AspNetCore Refpack Assemblies
```

Comment:
Who will keep this document up-to-date? Isn't this information already expressed in code? Ie we don't define such a table for Microsoft.NetCore.App as the code already expresses this well.

Reply:
>  Isn't this information already expressed in code?

We have this information in code for 6.0 but not for 5.0

> Who will keep this document up-to-date? 

for 5.0 it is fixed, for 6.0 the aspnetcore will start depending on the transport package so they will inform us if they need anything else in package.